docs/releases/2.0.txt

```diff
@@ -194,6 +194,10 @@ Models
 * The new :class:`~django.db.models.functions.StrIndex` database function
   finds the starting index of a string inside another string.
 
+* The :class:`~django.db.models.DecimalField` ``to_python()`` method now uses
```

Comment:
I don't think it's a feature worth mentioning considering the main benefit seems to be to handle programming errors (assigning floats to decimals).